### DISCUSSION TITLE
How do I completely reset the settings inside?

### DISCUSSION THREAD

**Discussion by Huangwwz** [T-Troll/alienfx-tools#368] [General]
Hello, 
The question I want to ask is, how do I completely uninstall the tool? The reason is this, I made a wrong setting, resulting in a strange error in the grid of the selection area (as shown in the figure). I tried to restore, but it didn't work. When I uninstall and reinstall again, it will always remember my settings before uninstalling. How can I do it? Sorry to bother you, looking forward to your reply.

**Comment by Huangwwz** [comment]
![image](https://github.com/T-Troll/alienfx-tools/assets/96620911/4c736337-a189-4d91-b52b-d05f6f86e9f0)

**Reply by T-Troll** [reply]
Run `alienfx-config delete` - this clean all configuration.

**Reply by Huangwwz** [reply]
Thank you for your reply!
I successfully restored the initial settings by deleting some residues in the registry, but I would like to ask if alienfx-config delete is in the installation directory? I only found the file alienfx-config.cmd.
Looking forward to your reply.

**Reply by T-Troll** [reply]
Yes, it is. It's a simple script to backup/restore/remove all configuration data.
And yes, it's into registry, and i don't remove it upon deinstallation - this will break app update

**Reply by Huangwwz** [reply]
I understand, thanks for your reply!

**Reply by sole4iman-ops** [reply]
@T-Troll Alienfx has messed up all my lights, how do i restore all setting to default? You mentioned Run alienfx-config delete is this on CMD or RUN? Could you please tell me step by step, thanks

**Reply by T-Troll** [reply]
In case you need to reset light device in itself, you don't need to clear app config. Just shut down (NOT sleep) the system, remove all cables including power cord, wait ~15 seconds, connect and run back. This should reset light devices state to default.

**Reply by sole4iman-ops** [reply]
@T-Troll i tried that, it doesn't work, lights are still messed up :( any other solution please?

**Reply by T-Troll** [reply]
Ok... First, what do you mean "messed up"? Please explain.

**Comment by sole4iman-ops** [comment]
![unnamed](https://github.com/user-attachments/assets/83876311-fbd5-4014-bb1b-7c658fd4de00)
Part of the ring changed to different color and i can't figure it out to undo this, it's also blinking. :(

**Reply by T-Troll** [reply]
Ohhh..... I write it into manual, guides and into the dialogue itself - NO ONE READ anyway.... I'm a bi tired about this.

DO NOT set "similar" system at the detection stage, but EXACTLY the same. And this is the consequences.

This light set as a hardware power button as a result. I hope you keep tools and configuration intact, in this case you can check lights into Device tab, found this one and uncheck "power button" checkbox. 

And DO NOT do this again!

**Reply by sole4iman-ops** [reply]
Doesn't work, is there anyway to remove this whole software with it's current config from the system?

**Comment by sole4iman-ops** [comment]
Doesn't work, is there anyway to remove this whole software with it's
current config from the system?

**Comment by 5x-Rhadamanthus** [comment]
I would like to hear a way to completely reset the configuration for the lighting due to this application. My system has random colors now due to the app mis-recognizing the LEDs.

**Reply by T-Troll** [reply]
Ohh... I need to increase the size of the warning (or maybe add a 30-second non-stop full-screen warning): DO NOT use "similar" or other model configs!
So... First, you need to check and remove the hardware power button if it has any. Select AW-ECC device, check all lights you have on it, UNCHECK "Power" checkbox if the light has it.
Second, for any device you have, select it and press the "Clear device" buttons. This removes all lights' data.
In case it's not enough, open CMD into the tools folder and run `alienfx-config delete` - this wipes all tools data.
